lucene/src/test/java/org/hibernate/hql/lucene/test/LuceneQueryParsingTestBase.java

```diff
@@ -111,24 +111,24 @@ public void shouldCreateQueryWithUnqualifiedPropertyReferences() {
 	public void shouldCreateNegatedQuery() {
 		assertLuceneQuery(
 				"from IndexedEntity e where NOT e.name = 'same'",
-				"-name:same *:*" );
+				"-name:same #*:*" );
```

Comment:
This "#" seems weird. Why did you add this? Where did you find documentation about this syntax?